### ISSUE TITLE
Error in the Vega - Sankey - IP/Firewall Blocks dashboard

### ISSUE THREAD

**Issue by abdeslammazouz** [wwalker0307/ElasticFlare#3]
Hello,

First of all, thank you for this great effort. 

I have an issue with the Vega - Sankey - IP/Firewall Blocks visualization. When I open the Blocks dashboard, I get this message at the visualization space:
Infinite extent for field "y1": [Infinity, -Infinity] 

Do you have an idea why?

Thank you

**Comment by wwalker0307** [comment]
I'm not quite sure.  It doesn't appear to be related to record count.  If you go into the visualization, lines 20 and 27 define the fields to aggregate.  These should be `client.ip` and `edge.firewall.rule.description.keyword`

**Comment by abdeslammazouz** [comment]
Yes. That is correct. Line 20 and 227 have the right values. I was doing some research and it might be related to @timestamp. 
However, I tried many combinations and none of them render the actual visualization.

**Comment by abdeslammazouz** [comment]
Unfortunately, i can see the field in the index pattern. But, I don't see it in the search at the Discover part. Maybe it has changed from cloudflare?
